File: C#_Projects/Magaza/Gıda.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Magaza
{
    class Gıda : Anasayfa
    {
        public int rafOmru;

        public Gıda(int kodu, string adi, float agirligi, string ulkesi) : base(kodu, adi, agirligi, ulkesi )
        {
        }

        public override void genelOzellikleriGoster()
        {
            Console.WriteLine($"Bu gıdanın ürün kodu: {urunKodu}, ürün adı {urunAdi}, ürün ağırlığı: {urunAgirligi} kilogram, urunun menşei ise {urunMensei} gözükmektedir");
        }

        public override void nasilSatilacak()
        {
            if (rafOmru <= 5)
            {
                Console.WriteLine("Bu gıda ürünü satılamaz, raf ömrü geçmek üzere");
            }
            else
            {
                Console.WriteLine("Bu gıda ürünü satılabilir, raf ömrü yeterli.");
            }
        }
    }
}

File: Javascript_Projects/Magaza/Gıda.js
import { Anasayfa } from "./Anasayfa.js";

export class Gıda extends Anasayfa {

    constructor(urunKodu, urunAdi, urunAgirligi, rafOmru) {
        super(...arguments);
        this.rafOmru = rafOmru;

    }

    genel_ozellikleri_goster() {
        console.log(`Bu gıdanın ürün kodu ${this.urunKodu}, ürün adı ${this.urunAdi} , ürün ağırlığı: ${this.urunAgirligi} kilogram, urunun menşei ${this.urununMensei}, ürünün raf ömrü ise ${this.rafOmru} gün olarak gözükmektedir`);
    }


    nasil_satilacak() {

        if (this.rafOmru <= 5) {
            console.log("Bu gıda ürünü satılamaz, ürünün raf ömrü geçmiş veya geçmek üzere");
        }
        else {
            console.log("Bu gıda ürünü satılabilir, raf ömrü yeterli.");
        }
    }
}